src/main/java/org/fundacionjala/sfdc/pageobjects/Contacts/SFCNewModifyPage.java

```diff
@@ -0,0 +1,88 @@
+package org.fundacionjala.sfdc.pageobjects.Contacts;
+
+import org.fundacionjala.sfdc.commons.DriverManager;
+import org.fundacionjala.sfdc.pageobjects.SalesForceConnection;
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class SFCNewModifyPage extends SalesForceConnection {
+    private WebDriverWait wait;
+    @FindBy(how = How.ID, using = "name_firstcon2")
+    private WebElement contactNameTextField;
+
+    @FindBy(how = How.ID, using = "name_lastcon2")
+    private WebElement contactLastNameTextField;
+
+    @FindBy(how = How.CSS, using = "#topButtonRow> input[name='save']")
+    private WebElement saveContactButton;
+
+    /**
+     * Constructor.
+     */
+    public SFCNewModifyPage() {
+        super();
+        PageFactory.initElements(this.webDriver, this);
+    }
+
+    /**
+     * Method to get "save" button.
+     * @return web element save button.
+     */
+    public WebElement getSaveContactButton() {
+        wait = DriverManager.getInstance().getWaiter();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2209)